aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
Why are you removing serializable? I believe this will break the code when it's using an actual distributed cache, as opposed to the in memory version used for testing.